0.Exploratory_Data_Analysis/nbconverted/Exploratory_Data_Analysis.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Exploratory Data Analysis on Wave Data
+
+# ### Wave1 dilated 50 data
+# Here we understand more about the data and how it is distrubuted.
+# Wave1 data are extracted features from raw images.
+# These images were processed via Cellprofiler pipelines
+#
+# Specifically wave1 is looking at Gasdermin-D and Nuclei Staining from a cell painting experiment.
+#
+# Further, nuclei were dilated using multiple values of pixel dilation. Here we use data for the 50 pixel dialation
+
+#
+# ### Wave3
+# Here we look into how the data are distributed in wave3, the same plate data as wave1 but with full cell painting dataset including the Gasdermin D channel for a total of 6 channels to analyze.
+# These data sets contain extracted features from raw image dataset. See README for information as to how raw image data was processed.
+
+# In[1]:
+
+
+import sys
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+get_ipython().run_line_magic("matplotlib", "inline")
+import itertools
+
+from sklearn.cluster import KMeans
+from sklearn.manifold import TSNE
+
+sys.path.append("..")
+from ..utils.utils import *
```

Comment:
Please refrain from using `*` imports. This can make it difficult to figure out which functions are being imported and making it hard to find within the code base. 

Also this can be dangerous because you are importing all these function names, and variable names into your code base. There can be instances if you have a function (or variable) that may have the same name and can be overwritten due to the import. 

Here's a nice [source](https://www.geeksforgeeks.org/why-import-star-in-python-is-a-bad-idea/)